salt/_modules/containerd.py

```diff
@@ -0,0 +1,60 @@
+'''
```

Comment:
For this kind of docstring, it's more common to use double quote instead of single quote (on the ring repo it's 1406 vs 29223 occurrences)

Reply:
https://docs.saltstack.com/en/latest/topics/development/conventions/style.html

Reply:
Ho, ok then: when in Rome, do as the Romans do.

I didn't know it was Salt style. Then it's correct.

Reply:
Yeah, based on formatting used in other Salt modules. However, if we decide to e.g. adopt PEP8 for this project (with proper linting), then I'll adjust accordingly.